src/Components/WelcomePage/WelcomePage.js

```diff
@@ -4,60 +4,96 @@ import Dimensions from 'Dimensions';
 import { ThemeProvider, Button } from 'react-native-elements';
 import { AddUser } from '../../Actions';
 import { connect } from 'react-redux';
+import { RalewayText, RalewayBoldText } from '../../Utilities/RalewayText';
 
 const DismissKeyboard = ({ children }) => (
   <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
     {children}
   </TouchableWithoutFeedback>
 );
 
+let themeColor = '#667D90';
+let accentOne = '#7C9DB1';
+let accentTwo = '#ACC6D0';
+let windowHeight = Dimensions.get('window').height;
+let windowWidth = Dimensions.get('window').width;
+
 const theme = {
+
   colors: {
-    primary: '#D3E2EB',
+    primary: accentOne,
   },
+
   Button: {
-    titleStyle: {
-      color: 'black',
-    },
-  }
+    titleStyle: { color: 'white', fontFamily: 'raleway' },
+    width: Dimensions.get('window').width - 50,
+  },
+
 }
 
 class WelcomePage extends Component {
-  static navigationOptions = {
-    header: null
-  }
-
   constructor(props) {
     super(props);
     this.state = {
-      userName: ''
+      userName: '',
+      showError: false
     }
   }
 
+  static navigationOptions = {
+    header: null
+  }
+
   handleChange = (name) => {
-    this.setState({ userName: name })
+    this.setState({ 
+      userName: name, 
+      showError: false 
+    });
   }
 
   handleSave = () => {
     const { userName } = this.state;
-    this.props.AddUser(userName);
-    this.props.navigation.navigate('homePage');
+
+    if (userName) {
+      this.props.AddUser(userName);
```

Comment:
I think that it might be best to use camel-case for 'AddUser' unless this is a React Native syntax that needs to be kept